assets/js/blocks/cart-checkout/cart/full-cart/checkout-button/american-express-logo.js

```diff
@@ -0,0 +1,29 @@
+export function AmericanExpressLogo( { className } ) {
```

Comment:
We might want to upstream this icon to `@automattic/composite-checkout`. But I think it might be better to postpone it for now until we implement the dynamic payment icons.

Reply:
Agreed.